dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -11,6 +11,13 @@
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
 
+# Eclipselink version info
+eclVersion=3.0.0
+#eclHash=69f2c2b
+#eclFullVersion=${eclVersion}.${eclHash}
+eclFullVersion=${eclVersion}
```

Comment:
Should eclFullVersion have a suffix like -M1 or -R1.5?

Reply:
The current version that was dropped recently is -R1.5 (I got an earful for using a local sandbox build while using the regular published maven coordinates, when I should have used an obvious temp coordinate, already).  When ECL 3.0 starts going live, it'll be version coordinates like 3.0.0, 3.0.1, etc.